llvm/test/CodeGen/AMDGPU/agpr-copy-no-free-registers.ll

```diff
@@ -366,9 +365,6 @@ define amdgpu_kernel void @no_agpr_no_reserve(ptr addrspace(1) %arg) #0 {
   ret void
 }
 
-; FIXME: This case is broken. The asm value passed in v32 is live
```

Comment:
Can you adjust the attributes to reproduce the previous situation? This isn't a proper fix for this